content/academy/apify_platform/getting_started/apify_api.md

```diff
@@ -44,11 +44,17 @@ https://api.apify.com/v2/acts/2bmkwWmp8fJkYzxqp/run-sync-get-dataset-items?token
 
 There are many more parameters which can be passed to this endpoint. You can learn about them [here](https://docs.apify.com/api/v2#/reference/actors/run-actor-synchronously-and-get-dataset-items/run-actor-synchronously-with-input-and-get-dataset-items)
 
-> Note: It is safer to put your API token in the **Authorization** header like so: `Authorization: Bearer YOUR_TOKEN`.
+> Note: It is safer to put your API token in the **Authorization** header like so: `Authorization: Bearer YOUR_TOKEN`. This is very easy to configure in [Postman]({{@link tools/postman.md}}), [Insomnia]({{@link tools/insomnia.md}}), or any other modern HTTP client.
 
 ## [](#sending-the-request) Sending the request
 
-Finally, we'll send the request:
+If you're not using an HTTP client, you can send the request through your terminal with this command:
+
+```curl
+curl -d '{"num1":1, "num2":8}' -H "Content-Type: application/json" -X POST "https://api.apify.com/v2/acts/2bmkwWmp8fJkYzxqp/run-sync-get-dataset-items?token=YOUR_TOKEN_HERE&format=csv"
```

Comment:
I deleted the actor, so it shouldn't matter

Reply:
It shouldn't, but still, you use the YOUR_TOKEN_HERE placeholder, so all pieces of the URL that should be replaced should have a placeholder. Otherwise the users will get confusing errors.

```suggestion
curl -d '{"num1":1, "num2":8}' -H "Content-Type: application/json" -X POST "https://api.apify.com/v2/acts/YOUR_USERNAME~adding-actor/run-sync-get-dataset-items?token=YOUR_TOKEN_HERE&format=csv"
```